### ISSUE TITLE
Changes to v-model not being reflected in multi-select

### ISSUE THREAD

**Issue by adam-hanna** [vuematerial/vue-material#575]
### Steps to reproduce
1. Navigate to this [codepen](http://codepen.io/adam-hanna/pen/wJKBav)
2. Wait the two seconds for the setTimeout to pop the two, pre-existing selections
3. Select "User 2" by clicking on the multi-select with your mouse

### Which browser?
Chrome 56

### What is expected?
"User 2" should be the only user showing

### What is actually happening?
The select is showing "User 1, User 2, User 3"

### Reproduction Link
http://codepen.io/adam-hanna/pen/wJKBav

**Comment by VicarEscaped** [comment]
Hi, @adam-hanna 
`User 1, User 2, User 3` is shown because it cached in **multipleOptions** variable.
So, do display properly just clear it, like this `this.$refs.multiselect.multipleOptions = {}`
There **multiselect** is _ref_ to mdSelect
[link](http://codepen.io/anon/pen/Mpaaeg)

**Comment by adam-hanna** [comment]
That did it, thanks!

Still seems a bit hacky, though. I would have expected it to match exactly the v-model, no?

**Comment by VicarEscaped** [comment]
> Still seems a bit hacky, though. I would have expected it to match exactly the v-model, no?

Yep, as i see, in **setTextAndValue**, which handle v-model changes, [no proper handler for multiple values](https://github.com/marcosmoura/vue-material/blob/master/src/components/mdSelect/mdSelect.vue#L129-L130).
I think, there is should be a check that mdSelect handle multiple values. May be somethind linke this
```js
if (this.multiple) {
  this.multipleOptions = output.value
}
```

**Comment by pablohpsilva** [comment]
I fix it via PR #749 .
Soon you'll have the bug fix @adam-hanna !

Thank you guys for opening this issue. :D
